### PR TITLE
Added notes to say that RFCs that currently use the new format should continue to do so

### PR DESCRIPTION
**Pull request by swcurran**

### CODE REVIEW

**Comment by swcurran** [comment]
@troyronda - please confirm that these changes address your concern.  If so, these will be merged and I'll submit a new PR to start the "Accepted" status change clock.

File: features/0348-transition-msg-type-to-https/README.md [hunk 1]
```diff
@@ -11,15 +11,16 @@
 ## Summary
 
 Per issue [#225](https://github.com/hyperledger/aries-rfcs/issues/225), the
-Aries community has agreed to change the prefix for protocol message types from
-`did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/` to `https://didcomm.org/`. Examples of the two message types forms are:
+Aries community has agreed to change the prefix for protocol message types that currently use the
```
Comment (troyronda) [inline features/0348-transition-msg-type-to-https/README.md]: I also think newly written RFCs should also use the new format. i.e., we shouldn't need to transition new efforts.

**Comment by troyronda** [comment]
@swcurran Thanks for the update! I think the new text nicely addresses the already existing RFCs. I would also suggest that new RFCs should also just start with the new format (commented inline).

**Comment by swcurran** [comment]
Good catch @troyronda - added.

**Comment by dhh1128** [comment]
I'm cool with merging this.

**Pull request merged**